test/travis_script.sh

```diff
@@ -31,23 +31,39 @@ mysql -u swefreq -h 127.0.0.1 -P 3366 swefreq_test < test/data/load_dummy_data.s
 
 
 echo ">>> Test 3. Check that the backend starts"
-cd backend
 
+cd backend
 ../test/01_daemon_starts.sh
+cd ..
```

Comment:
Use `(cd backend && ../test/01_daemon_starts.sh)` instead of changing directory back and forth.